Calling/ClientApp/src/components/ChatThread.tsx

```diff
@@ -302,6 +307,22 @@ export default (props: ChatThreadProps): JSX.Element => {
     setMessagesWithAttachedRef(newMessagesWithAttached);
   };
 
+  const isUploadedFileMessage = (messageContent: string): Boolean => {
```

Comment:
Instead of a `Boolean`, if this returned the parsed JSON (maybe a `FileMessage` object or `null`), then you could avoid re-parsing it later.

Reply:
fixed